mealsharingdotnet/Program.cs

```diff
@@ -0,0 +1,131 @@
+
+using System.Linq;
+using System.Text.Json;
+using Microsoft.AspNetCore.Mvc;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddScoped<IMealService, FileMealService>();
+builder.Services.Remove(ServiceDescriptor.Transient<IMealService, FileMealService>());//delete meal service
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+var app = builder.Build();
+if (app.Environment.IsDevelopment())
+{
+  app.UseSwagger();
+  app.UseSwaggerUI();
+}
+
+app.MapGet("/", () => "week5-mealsharing dotnet");
+
+app.MapPost("/meals", ([FromServices] IMealService mealSharingService, Meal meal) =>
+{/*eg: postman input
```

Comment:
Nice, I would maybe add this on the top of the method and use [C# documentation comments](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments)

eg
```csharp
/// <summary>
/// Endpoint for adding a new blog post.
/// <example>
/// postman input
/// <code>
/// {
///     "headline": "Nepal Hot Pot",
///         "imageURL": "https://askme.com",
///         "bodyText": "popular dumpling",
///         "location": "ishoj,cph",
///         "price": 500
/// }
/// </code>
/// </example>
/// </summary>
app.MapPost("/meals", ([FromServices] IMealService mealSharingService, Meal meal) =>
{
 ....
})
```

Reply:
ok. thank you so much !